docs/modules/deployment/pages/time-series-storage/newts/newts.adoc

```diff
@@ -1,71 +1,99 @@
 
 [[newts-configure]]
-= Configure {page-component-title} Core to Use Newts
+= Configure {page-component-title} to Use Newts
 
-.Create a configuration file with the time series database settings
+This section describes how to configure your {page-component-title} instance to use Newts.
+
+== Setup
+
+Follow these steps to set up Newts on your {page-component-title} instance:
+
+. Create a configuration file with your time series database settings:
++
 [source, console]
-----
 sudo vi etc/opennms.properties.d/timeseries.properties
-----
 
-.Configure Newts as the time series strategy
+. Configure the storage strategy:
++
 [source, properties]
 ----
-# Configure storage strategy
-org.opennms.rrd.storeByForeignSource=true<1>
-org.opennms.timeseries.strategy=newts<2>
+org.opennms.rrd.storeByForeignSource=true <1>
+org.opennms.timeseries.strategy=newts <2>
+----
+<1> Associate time series data using the foreign source and ID instead of the database-generated node ID.
+<2> Set time series strategy to use `newts`.
+If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing installation and want to keep historical metrics, do not set the `org.opennms.timeseries.strategy` to `newts` until you have written enough data to expire any collected data prior to enabling the dual write feature.
```

Comment:
```suggestion
If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing installation and want to keep historical metrics, do not set the `org.opennms.timeseries.strategy` to `newts` until you have written enough data to expire any collected data prior to enabling the dual write feature.
```
The last part of the sentence doesn't quite parse. Is it to expire the data that was collected before enabling the dual write feature, or don't do this until you make sure you've expired the data, then enable the dual write feature?

Reply:
I think it's the latter, but I'll defer to @jeffgdotorg for confirmation.

Reply:
Everything looks good to me. Will approve once we hear back from Jeff about the above comment.

Reply:
I'm reasonably confident it's meant to be the latter.

Reply:
@jeffgdotorg, does this work better? Is there any more specific information that you think would make the sentence parse more naturally?

```suggestion
If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing {page-component-title} installation and you want to keep historical metrics, make sure that the written data has expired before you set `org.opennms.timeseries.strategy` to `newts`.
```

Reply:
This works for me!